docs/elements/elements.md

```diff
@@ -0,0 +1,280 @@
+        },
+
+        // Destructore, here set to the jQuery empty function
+        destroy: $.noop,
+
+        // Public function, callable like "$('#element-13').mbMyClass('methodA', parameterA, parameterB)"
+        methodA: function(parameterA, parameterB) {
+            this._methodB(parameterA);
+        },
+
+        // Private function, only callable from within this widget
+        _methodB: function(parameterA) {
+            // The triggered signal will be named "mbmyclassmagicdone" (all lowercase)
+            this._trigger('magicdone');
+        },
+
+        _onMagicDone: function() {
+            alert("Oh, magic!");
+        },
+
+        _clickCallback: function(event) {
+            var target = $(event.target);
+            var id = target.attr('id');
+            // ...
+        }
+
+    });
+
+    })(jQuery);
+```
+
+For event handling, jQuery.proxy is your friend to ensure that the callback is guaranteed to be executed in the right context:
+
+```javascript
+    // ...
+
+    this.element.click($.proxy(this._clickCallback, this));
+
+    // ...
+```
+
+This way, `this` inside the clickCallback method is the this given as the second parameter here (usually the widget instance) and not the HTML element which triggered the event. To access the HTML event, use the target property of the event passed as the argument to the clickCallback method.
+
+## Element to Element communication
+
+There's an active and a passive way to communicate with another widget. The active way is to call a public method of the other widget. For that, you have to select the widget's HTML element with jQuery and call the method like this:
+
+```javascript
+    var otherElement = $('#element-13').mbMyClass('methodA', parameterA, parameterB);
+```
+
+This is standard jQuery UI stuff and pretty much self-explanatory. The more tricky question is how do you know the other's HTML element? As you can see, using the ID to select the element is preferred, but these IDs are generated on the fly by Mapbender when the application is started, so you can't assume that the ID is the always the same.
+Luckily you can pass an element ID in the configuration as the target options for an Element. This will be replaced with the run-time ID of that target Elements' HTML element for you, so that in your widget code you can access the right ID as `this.options.target`.
+
+```javascript
+    $('#' + this.options.target).mbMyClass('methodA', parameterA, parameterB);
+
+```
+
+The passive way for communication is to subscribe to events of another target. You also need to know the HTML element,
+but you can now listen for the other widget to call your widget. This is done using default jQuery events. <!--Well, almost
+default jQuery events: If you use the "_trigger" method provided by the jQuery UI widget factory-->
+
+## How to create your own Element?
+
+Mapbender offers an app/console command to create different elements:
```

Comment:
bin/console 
Is the command to create an element still working? Else delete this section